cmd/kubeadm/app/constants/constants.go

```diff
@@ -138,6 +139,9 @@ var (
 	// DefaultTokenUsages specifies the default functions a token will get
 	DefaultTokenUsages = []string{"signing", "authentication"}
 
+	// DefaultTokenGroups specifies the default groups a token will get
+	DefaultTokenGroups = []string{bootstrapapi.BootstrapDefaultGroup}
```

Comment:
The fewer imports that kubeadm has on k8s.io/kubernetes the easier it's going to be split out. Constants seem like a good candidate for things to just copy/paste and not add another import.